build/cake/build-and-package.cake

```diff
@@ -10,6 +10,7 @@ Task ("nuget")
         .EnableBinaryLogger ($"./output/nuget.{CONFIGURATION}.binlog")
         .WithProperty ("NoBuild", "true")
         .WithProperty ("PackageOutputPath", MakeAbsolute ((DirectoryPath)"./output/").FullPath)
+        .WithProperty ("PackageVersionSuffix", NUGET_VERSION_SUFFIX)
```

Comment:
There could be a few random projects that need `$(PackageVersionSuffix)`:
* https://github.com/dotnet/android-libraries/pull/1079/files#diff-62f40b2ecf75c94950e34373da2b9ec3f93f231c4e294f4076316395c2e91e3cR36

And then how do we update the packages dependences like:
```xml
  <ItemGroup>
    <PackageReference Include="Xamarin.AndroidX.Emoji2" Version="[1.2.0.2, 1.2.1)" PrivateAssets="none" />
  </ItemGroup>
```

I remember doing some find/replace in `generated\*\*.csproj` to update this to:
```xml
  <ItemGroup>
    <PackageReference Include="Xamarin.AndroidX.Emoji2" Version="1.2.1$(PackageVersionSuffix)" PrivateAssets="none" />
  </ItemGroup>
```

Maybe we should wait, and try fixing `$(PackageVersionSuffix)` in a future PR?